chia/cmds/plotnft_funcs.py

```diff
@@ -63,9 +63,14 @@ async def create_pool_args(pool_url: str) -> dict[str, Any]:
 
 
 async def create(
-    wallet_rpc_port: Optional[int], fingerprint: int, pool_url: Optional[str], state: str, fee: uint64, *, prompt: bool
+    rpc_info: NeedsWalletRPC,
```

Comment:
Don't really love this hint existing outside the class infrastructure.  Maybe you can enter the context manager outside this function call and pass in the `wallet_info` object?

Reply:
not sure I follow this one

Reply:
I see `NeedsWalletRPC` as only being used on a `@chia_command` object. Once it is applied to something else, it makes less sense.  I think in the `.run` function for these, you should do the `async with <whatever> as wallet_info:` and then pass `wallet_info` into these functions.